src/app/Layout/Layout.tsx

```diff
@@ -1,9 +1,20 @@
 import { Outlet } from 'react-router-dom';
+import { AppSidebar } from './AppSidebar';
+import { SidebarProvider } from '@/components/ui/sidebar';
+import { Navbar } from './Navbar';
+import { Footer } from './Footer';
 
 export const Layout = () => {
   return (
-    <div>
-      <Outlet />
-    </div>
+    <SidebarProvider>
+      <AppSidebar />
+      <main className="flex w-full flex-col">
+        <div className="flex-grow">
+          <Navbar />
```

Comment:
`Navbar` is a bit misleading. In your case this is just a header, the navbar is the sidebar.